NonInteractivePatGenerationSample/NonInteractivePatGenerationSample/Program.cs

```diff
@@ -41,21 +42,35 @@ public static void Main(string[] args)
             var vstsCredential = new VssAadCredential(token);
 
             var connection = new VssConnection(new Uri(azureDevOpsOrganizationUrl), vstsCredential);
-            var client = connection.GetClient<Microsoft.VisualStudio.Services.DelegatedAuthorization.WebApi.TokenHttpClient>();
 
-            var sessionToken = new SessionToken
+            CreatePatUsingNewPatApi(connection);
+        }
+
+        /// <summary>
+        /// Creates a PAT using the PAT Lifecycle Management API (https://learn.microsoft.com/en-us/rest/api/azure/devops/tokens)
+        /// </summary>
+        /// <param name="connection">Azure DevOps connection</param>
+        /// <returns></returns>
+        private static void CreatePatUsingNewPatApi(VssConnection connection)
```

Comment:
Why not just "CreatePat"? By now there's nothing new about this API anyway ;)

Reply:
Changed. Originally I was thinking of leaving the old sample for history purposes.